bitnami/postgresql-ha/templates/pgpool/deployment.yaml

```diff
@@ -65,6 +65,10 @@ spec:
             {{- end }}
             - name: PGPOOL_ENABLE_LOAD_BALANCING
               value: {{ ternary "yes" "no" .Values.pgpool.useLoadBalancing | quote }}
+            {{- if .Values.pgpool.useLoadBalancing }}
+            - name: PGPOOL_DISABLE_LOAD_BALANCE_ON_WRITE
```

Comment:
The env var is called `PGPOOL_DISABLE_LOAD_BALANCE_ON_WRITE` and the parameter `pgpool.disableLoadBalancingOnWrite`, that seems to be contradictory

Reply:
Yes, your right! changing it!